Library/Homebrew/tap.rb

```diff
@@ -847,6 +847,22 @@ def tap_migrations
     end
   end
 
+  # Array with autobump names
+  sig { returns(T::Array[String]) }
+  def autobump
+    @autobump ||= if (autobump_file = path/HOMEBREW_TAP_AUTOBUMP_FILE).file?
+      autobump_file.readlines(chomp: true)
+    else
+      []
+    end
+  end
+
+  # Whether this {Tap} allows running bump commands on the given {Formula} or {Cask}.
+  sig { params(formula_or_cask_name: String).returns(T::Boolean) }
+  def allow_bump?(formula_or_cask_name)
+    ENV["HOMEBREW_TEST_BOT_AUTOBUMP"].present? || !official? || autobump.exclude?(formula_or_cask_name)
```

Comment:
We could maybe just read `GITHUB_ACTOR` or similar here to avoid having to carry around an extra environment variable in our workflows.